src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs

```diff
@@ -329,8 +329,16 @@ static string GetTargetAssembly (ITaskItem assemblyName)
 				string assetsDir = Path.Combine (importsDir, "assets");
 
 				var stamp = new FileInfo (Path.Combine (outdir.FullName, Path.GetFileNameWithoutExtension (aarFile.ItemSpec) + ".stamp"));
-				if (stamp.Exists && stamp.LastWriteTimeUtc > new FileInfo (aarFile.ItemSpec).LastWriteTimeUtc)
+				if (stamp.Exists && stamp.LastWriteTimeUtc > new FileInfo (aarFile.ItemSpec).LastWriteTimeUtc) {
+					if (Directory.Exists (resDir))
+						resolvedResourceDirectories.Add (new TaskItem (resDir, new Dictionary<string, string> {
+							{ OriginalFile, Path.GetFullPath (aarFile.ItemSpec) },
+							{ SkipAndroidResourceProcessing, "True" },
+						}));
```

Comment:
I think `resolvedResourceDirectories` is a `List<string>` on 15.9, so this would work:
```csharp
resolvedResourceDirectories.Add (resDir);
```
The bug fixed here is that `resolvedResourceDirectories` and `resolvedAssetDirectories` are missing this item.